lib/deploy/deploy-support.js

```diff
@@ -291,15 +291,26 @@ async function makeService({
       service = await fc.getService(serviceName);
     } catch (ex) {
       
-      if (ex.code === 'AccessDenied' || ex.code === 'ERR_INVALID_CHAR' || ex.code === 'ENOTFOUND') {
-        console.error(red(`The accountId entered is incorrect. We speculate that you used the subuser uid, login name, login name prefix, display name or thers\n`+
-        `please enter the correct accountId, which is the account ID under your primary account. You can reset your account ID by executing fun config\n`+
-        `For the method of taking the primary account ID, please refer to https://account.console.aliyun.com/#/secure`));
+      if (ex.code === 'AccessDenied') {
+
+        if (ex.message.indexOf('the caller is not authorized to perform') !== -1) {
+
+          console.error(red(`You have not given sub-account permission to manage the fc service, The permission link is https://ram.console.aliyun.com/users`));
+
+        } else if (ex.message.indexOf('FC service is not enabled for current user') !== -1) {
+
+          console.error(red(`You have not opened the function computer service, the opening address is https://www.aliyun.com/product/fc`));
```

Comment:
FC service is not enabled for current user. Please enable FC service before using fun. 
You can enable FC service on this page https://www.aliyun.com/product/fc .